### DISCUSSION TITLE
Import existing nginx .conf files

### DISCUSSION THREAD

**Discussion by rosskinard** [NginxProxyManager/nginx-proxy-manager#1356] [Q&A]
I'm just setting up an UnRAID server and I've been moving assorted services and functions from multiple machines into one. I currently have a Docker container running nginx and have all my configs there + wildcard SSL cert, etc.

Is there an easy way to import the existing .conf files, logs and certs?

Thanks
Ross

**Comment by th0mcat** [comment]
I would very much like to know this as well.

**Comment by BenLangers** [comment]
+1

**Comment by mattiasghodsian** [comment]
+1

**Comment by pr0fsmith** [comment]
Did anyone ever get an answer to this?
